example/contexts/JellyfishExampleContext.tsx

```diff
@@ -55,9 +58,11 @@ const JellyfishExampleContext = React.createContext<
   | undefined
 >(undefined);
 
+const isIosEmulator = Platform.OS === 'ios' && !Device.isDevice;
```

Comment:
nit: `isIosSimulator` (there is small difference between emulators and simulators)